src/@next/Pagination/paginationUtils.ts

```diff
@@ -9,6 +11,14 @@ export type PaginationProps = {
   onPageChanged?: (currentPage: number) => void;
 };
 
+export type IconOnlyPaginationProps = {
```

Comment:
the label property on Storybook is not working and thus I cannot see how a label between buttons looks like. Could you help to fix that so that I can check the style?

Reply:
yeah, forgot to handle the label property after changing the label type. will fix this one